src/components/calcite-input-time-picker/calcite-input-time-picker.tsx

```diff
@@ -0,0 +1,311 @@
+import { parseTimeString, Time, validateTimeString } from "../../utils/time";
+import { Theme } from "../interfaces";
+
+@Component({
+  tag: "calcite-input-time-picker",
+  styleUrl: "calcite-input-time-picker.scss",
+  scoped: true
+})
+export class CalciteInputTimePicker {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteInputTimePickerElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** The disabled state of the time input */
+  @Prop({ reflect: true }) disabled?: boolean = false;
+
+  /** Format of the hour value (12-hour or 24-hour) (this will be replaced by locale eventually) */
+  @Prop({ reflect: true }) hourDisplayFormat: "12" | "24" = "12";
+
+  /** aria-label for the hour input */
+  @Prop() intlHour?: string;
+
+  /** aria-label for the hour down button */
+  @Prop() intlHourDown?: string;
+
+  /** aria-label for the hour up button */
+  @Prop() intlHourUp?: string;
+
+  /** aria-label for the meridiem (am/pm) input */
+  @Prop() intlMeridiem?: string;
+
+  /** aria-label for the meridiem (am/pm) down button */
+  @Prop() intlMeridiemDown?: string;
+
+  /** aria-label for the meridiem (am/pm) up button */
+  @Prop() intlMeridiemUp?: string;
+
+  /** aria-label for the minute input */
+  @Prop() intlMinute?: string;
+
+  /** aria-label for the minute down button */
+  @Prop() intlMinuteDown?: string;
+
+  /** aria-label for the minute up button */
+  @Prop() intlMinuteUp?: string;
+
+  /** aria-label for the second input */
+  @Prop() intlSecond?: string;
+
+  /** aria-label for the second down button */
+  @Prop() intlSecondDown?: string;
+
+  /** aria-label for the second up button */
+  @Prop() intlSecondUp?: string;
+
+  /** The name of the time input */
+  @Prop() name?: string;
+
+  /** The scale (size) of the time input */
+  @Prop({ reflect: true }) scale: "s" | "m" | "l" = "m";
+
+  /** number that specifies the granularity that the value must adhere to */
+  @Prop({ reflect: true }) step = 60;
+
+  /** The color theme of the time-picker */
+  @Prop({ reflect: true }) theme: Theme;
+
+  /** The selected time */
+  @Prop({ reflect: true, mutable: true }) value?: string;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  //--------------------------------------------------------------------------
+
+  private calciteInputEl: HTMLCalciteInputElement;
+
+  private referenceElementId: string = guid();
+
+  //--------------------------------------------------------------------------
+  //
+  //  State
+  //
+  //--------------------------------------------------------------------------
+
+  @State() open = false;
```

Comment:
Should a user be able to start the component open?

Reply:
Probably?  I could add a prop for that, something like `initialOpen`.

Reply:
How about `open`? or `active`? Whatever we are consistent with other components.

Reply:
I like `open` since that's the prop calcite-popover uses.

Reply:
I think dropdown, input-date-picker, use `active`. We should be consistent here.